charters/DataPortal/charter.md

```diff
@@ -0,0 +1,80 @@
+# Data Portal
+
+
+## Description
+The Data Portal is a landing site for the Human Cell Atlas (HCA) Data Coordination Platform (DCP). It is hosted at https://www.data.humancellatlas.org/ with an access link from the main HCA website at https://www.humancellatlas.org/. It provides the user experience for scientific and general users to upload, explore, search, and run analyses on human genomic data. The Data Browser ("Explore") application provides a faceted search capability to find and download subsets of this data.
+
+## Objective
+The objective of the Data Portal is to provide the DCP website implementation. Provide the access point for scientific and general users to search all HCA genetic data in order to identify a subset of the data for download and/or further analysis, providing file/bundle search on multiple clouds at petabyte scale with public APIs.
```

Comment:
Is there a difference between scientific users and general users? is it worth making that distinction?

Reply:
Also, the HCA won't just distribute genetic data. I would remove that word. There will be proteomics, immunohistochemistry and maybe even MRIs plus whatever else gets thought up in the next few years.

If you want some examples I would change it to something like

"search all HCA data such as scRNA-seq and imaging transcriptomic data"

But removing the word genetic works too